Makefile

```diff
@@ -139,13 +139,13 @@ checks: check-style check-camel-case
 
 # automatically make python files pep 8-compliant
 fix-style:
-	pip3 install -r requirements/dev.txt -U
-	isort -rc $(MRLN)
-	isort -rc $(TEST)
-	isort *.py
-	black --target-version py36 $(MRLN)
-	black --target-version py36 $(TEST)
-	black --target-version py36 *.py
+	. $(VENV)/bin/activate; \
+	isort --line-length $(MAX_LINE_LENGTH) $(MRLN); \
```

Comment:
Added `--line-length` so that there was uniform convention across linting - previously Flake8 and PyLint were run at the configured values (127 chars/line), but isort and black went to their defaults (80 and 88, I think), which meant A. when manually invoked, a ton of files that passed under the CI would be changed and have to be reviewed when develop went into main B. the two would revert the other over any lines of 80 > length < 88, and so would've always failed out in the CI.